ocrolib/common.py

```diff
@@ -11,97 +11,32 @@
 import warnings
 import inspect
 import glob
+import pickle
 import cPickle
+from ocrolib.exceptions import (BadClassLabel, BadInput, FileNotFound,
+                                OcropusException)
 
 import numpy
-from numpy import *
+from numpy import (amax, amin, array, bitwise_and, clip, dtype, mean, minimum,
+                   nan, ndarray, sin, sqrt, zeros)
 import pylab
-from pylab import imshow
-from scipy.ndimage import morphology,measurements
+from pylab import (clf, cm, ginput, gray, imshow, ion, subplot, where)
```

Comment:
You mean https://github.com/QuLogic/ocropy/commits/standard-import? Sure, open a PR with the pertinent commits, looks really helpful, I'll happily merge/cherry-pick.